1) Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16.Найти доверительный интервал для оценки математического ожидания с надежностью 0.95,если выборочная средняя M = 80, а объем выборки n = 256.

In [2]:
import math
d=1.96*16/math.sqrt(256)
d

1.96

In [ ]:
Доверительный интервал [80-1.96, 80+1.96] 

2) В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,получены опытные данные:6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

3,4 задачи решать через тестирование гипотезы

In [3]:
M=(6.9+ 6.1+6.2+6.8+7.5+6.3+6.4+6.9+6.7+6.1)/10
M

6.589999999999999

In [6]:
cr=((6.9-M)**2+ (6.1-M)**2+(6.2-M)**2+(6.8-M)**2+(7.5-M)**2+(6.3-M)**2+(6.4-M)**2+(6.9-M)**2+(6.7-M)**2+(6.1-M)**2)/(10-1)
cr=math.sqrt(cr)
cr

0.4508017549014448

In [7]:
d=2.262*cr/math.sqrt(10)
d

0.32246174408757394

In [ ]:
Доверительный интервал [6.59-0.322, 6.59+0.322] 

3) Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

In [10]:
zp=(17.5-17)/(math.sqrt(4)/math.sqrt(100))
zp

2.5

In [ ]:
z для 0.95 табличное 1.65 <2.5, значит гипотеза верна.

4) Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек.
Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно,
что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [11]:
M0=200
M1=(202+203+199+197+195+201+200+204+194+190)/10
M1

198.5

In [12]:
cr=((202-M1)**2+(203-M1)**2+(199-M1)**2+(197-M1)**2+(195-M1)**2+(201-M1)**2+(200-M1)**2+(204-M1)**2+(194-M1)**2+(190-M1)**2)/(10-1)
cr=math.sqrt(cr)
cr

4.453463071962462

In [14]:
tp=(M1-M0)/(cr/math.sqrt(10))
tp

-1.0651074037450896

tt=3.25 Гипотеза верна при tp принадлежащему интервалу [-3.25;3.25]. -1.065 входит в указанный интервал, значит гипотеза верна.